airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -155,23 +164,44 @@ class Config:
         default_factory=lambda: datetime.now(tz=timezone.utc),
     )
 
-    include_deleted: bool = Field(
-        title="Include Deleted Campaigns, Ads, and AdSets",
+    campaign_statuses: Optional[List[ValidCampaignStatuses]] = Field(
+        title="Campaign Statuses",
         order=4,
-        default=False,
-        description="Set to active if you want to include data from deleted Campaigns, Ads, and AdSets.",
+        description="Select statuses you want to to be loaded in the stream. Empty means all available.",
+        default=[],
+    )
+
+    adset_statuses: Optional[List[ValidAdSetStatuses]] = Field(
+        title="AdSet Statuses",
+        order=5,
+        description="Select statuses you want to to be loaded in the stream. Empty means all available.",
+        default=[],
+    )
+
+    ad_statuses: Optional[List[ValidAdStatuses]] = Field(
+        title="Ad Statuses",
+        order=6,
+        description="Select statuses you want to to be loaded in the stream. Empty means all available.",
+        default=[],
+    )
+
+    adcreative_statuses: Optional[List[ValidAdCreativeStatuses]] = Field(
+        title="AdCreative Statuses",
+        order=7,
+        description="Select statuses you want to to be loaded in the stream. Empty means all available.",
+        default=[],
```

Comment:
This fields should be enum as we know full list of statuses + `ALL` status as default.

Reply:
It is list with fixed values - showed in UI as
<img width="873" alt="image" src="https://github.com/airbytehq/airbyte/assets/1858824/a01003c6-4322-4e62-857b-ee5bfbc71047">

Why it should be enum?

Default is empty field - there is no reason to set ALL and then replace in code to empty.

Reply:
Lose this lines. Thanks. 

The empty string flag as a trigger for syncing all statuses still confuses me. Better to add `All` and leave it as default. It will be more intuitive.

Reply:
Lose this lines. Thanks.

The empty string flag as a trigger for syncing all statuses still confuses me. Better to add All and leave it as default. It will be more intuitive.

Reply:
Added ALL statuses as default
<img width="503" alt="Pasted Graphic" src="https://github.com/airbytehq/airbyte/assets/1858824/88e64b90-4ff3-4e55-aaa8-585e742af366">